content/en/blog/2023/steering-contribution-seat-results/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Announcing the Contribution Seat holders for 2023"
+description: The Istio Steering Committee welcomes contributors from Google, IBM, Huawei and Red Hat.
+publishdate: 2023-02-06
+attribution: "Faseela K, for the Istio Steering Committee"
+keywords: [istio,steering,governance,community,contributor]
+---
+
+The [Istio Steering Committee](https://github.com/istio/community/blob/master/steering/README.md) consists of 9 Contribution Seats, proportionally allocated based on corporate contributions to the project, and 4 elected Community Seats.
+
+Last year, we [elected four members](/blog/2022/steering-election-results/) to the community seats. It's now time to announce the companies who fuel our growth by selecting the Contribution Seat members. As per the [Steering charter](https://github.com/istio/community/blob/master/steering/CHARTER.md), every February we look at which companies have made the most contributions to Istio based on an [annually agreed metric](https://github.com/istio/community/blob/master/steering/CONTRIBUTION-FORMULA.md).
+
+According to our [seat allocation process](https://docs.google.com/spreadsheets/d/1IIms6OT1DgJ_lbn5433sy5wvqNBIIQgQHoWZvpShSXk/edit#gid=1365082320), this year Google will be allocated 5 seats and IBM/Red Hat will be allocated 2. As the third largest contributor to Istio in the last 12 months, we are pleased to announce that Huawei has earned two Contribution Seats. 
```

Comment:
```suggestion
According to our [seat allocation process](https://docs.google.com/spreadsheets/d/1IIms6OT1DgJ_lbn5433sy5wvqNBIIQgQHoWZvpShSXk/edit#gid=1365082320), this year Google will be allocated 5 seats and IBM/Red Hat will be allocated 2. As the third largest contributor to Istio in the last 12 months, we are pleased to announce that Huawei has earned two Contribution Seats.
```

Reply:
markdown lint makes me sad.  trailing spaces don't offend anyone!

Reply:
> markdown lint makes me sad. trailing spaces don't offend anyone!

We could put an exception in the config if it makes sense to do so.

Reply:
It wouldn't generally be a big deal, because I would normally have access to accept my own or other people's suggestions/edit a PR before it went live, and I would just fix it.  I can't do that with Faseela's, for some reason. (Perhaps I am not as much of an admin as I was once.)

Can you accept the change?

Reply:
I didn't see the option to accept yours, so tried creating one of my own to see about accepting that. Doesn't appear to be an option anymore. I suspect there might have been some tightening of things as I noted I couldn't push a commit on another PR which I thought I could do earlier. It's probably a good idea to not allow anyone to necessarily push PRs. Might also be a setting on the fork?